dev_guide/service_accounts.adoc

```diff
@@ -170,6 +170,15 @@ To allow a secret to be mounted by a service account's pods, run:
 $ oc secrets link --for=mount <serviceaccount-name> <secret-name>
 ----
 
+[NOTE]
```

Comment:
Ah, I hadn't spotted the wording change (so IMO it's not clear enough ;).  Now I re-read it, I'm concerned that the following statement is actually not true and doesn't make sense:

> unless `serviceAccountConfig.limitSecretReferences` is set to `true` in the master configuration file, mounting secrets to a service account's pods will do nothing.

Something like the following would make sense:

if the admin has set limitSecretReferences to false (default), it is not required to use oc link --for=mount to enable use of a mountable secret by a service account.  However oc link --for=pull is always required to enable use of an image pull secret, regardless of the value of limitSecretReferences.